### PR TITLE
Add family test tile on homescreen (EXPOSUREAPP-12314)

### PR DESCRIPTION
**Pull request by SamuraiKek**
Added a new tile for the family test. 
Tried my best to center the text inside the badge, but it still looks off. Any suggestions are welcome.

Screen Shot 2022-03-18 at 17 10 32

### CODE REVIEW

**Review by CV113 — DISMISSED** [review]
UA approved

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragment.kt [hunk 1]
```diff
@@ -258,6 +258,9 @@ class HomeFragment : Fragment(R.layout.home_fragment_layout), AutoInject {
                 HomeFragmentDirections.actionMainFragmentToFederalStateSelectionFragment()
             )
             is HomeFragmentEvents.DeleteOutdatedRAT -> viewModel.deleteCoronaTest(event.identifier)
+            is HomeFragmentEvents.GoToFamilyTests -> {
+                // TODO: add wiring to family test results screen
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragment.kt:262]: I think you can do that now, a blank screen is there ,but it should help in this case

**Review by mtwalli — APPROVED** [review]
LGTM

**Review by chiljamgossow — APPROVED** [review]
LGTM

**Pull request merged**